waspc/cli/Main.hs

```diff
@@ -84,6 +86,25 @@ printUsage =
 printVersion :: IO ()
 printVersion = putStrLn $ showVersion version
 
+-- dynamically generate available option for wasp bash completion script tools/bash-completion
```

Comment:
We should probably separate this bash completion logic to separate file, since there is some of it now?

Reply:
I placed it in `Main.hs` in the first place because all wasp command options are defined there. I think it would be easier to manage that way.

Reply:
Some of them are here, but mostly the smaller ones, and those should probably also be separated (at least some of them).
Since we have a couple functions now that are used for completion, I would say it is complex enough to separate in a standalone file/module.